src/main/java/org/example/services/PolicyCreator.java

```diff
@@ -29,13 +32,13 @@ private Policy createPolicy(Map<String, String> questionAndAnswers, double polic
         String postcode = questionAndAnswers.get("postcode");
         String startdate = questionAndAnswers.get("policyStartDate");
 
-        if (questionAndAnswers.get("policy_type").equals("1")) {
+        if (questionAndAnswers.get("policy_type").equals(BRONZE.getValue())) {
```

Comment:
This is ok for now - but it would be good to set the answer within questionAndAnswers to be the actual enum type (so in this case "BRONZE"). But don't worry about that for now, what you have here is good - this is just something to think about.